server/lib/statsd-collector.js

```diff
@@ -65,10 +75,34 @@ function getGenericTags(body) {
     }
   }
 
+  // collect AB choices
+  if (body.ab) {
+    var abTags = [];
+
+    body.ab.forEach(function (item) {
+      if (item.event && item.event === 'choice' && item.experiment && item.choice) {
+        var tagName = 'ab_' + _normalizeTag(item.experiment);
+        var hadChoice = false;
+
+        Object.keys(item.choice).forEach(function (choiceKey) {
+          hadChoice = true;
+          tagName += '_' + _normalizeTag(choiceKey);
```

Comment:
we cannot use `:` sadly, that will mean it is a value and the DataDog interface will not let us query it. basically if you query `ab_` tags then you should have the experiment repository open to understand what those tags mean.